### ISSUE TITLE
Joke cut off

### ISSUE THREAD

**Issue by Redstone-Hamster** [poojarathore30/make-me-laugh#1]
Could you please make the jokes so they don't cut off?

**Comment by NChemdok** [comment]
yes please increase the no of char that get returned. Once the joke is more than you specified length... it just leaves it incomplete.
